managed/testdata/pg/certs/pmm-managed.cnf

```diff
@@ -0,0 +1,10 @@
+[req]
```

Comment:
Can you please add instructions how to generate the certificates from the config files?
This will be helpful later when no one remembers :)

Reply:
makefile with instructions added